db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -3404,6 +3404,11 @@ Status DBImpl::BackgroundCompaction(bool* made_progress,
       InstallSuperVersionAndScheduleWork(c->column_family_data(),
                                          &job_context->superversion_contexts[0],
                                          *c->mutable_cf_options());
+      // When the compaction succeeds, update the compact cursor using
+      // pre-computed next_compact_cursor_by_level_ (just for kRoundRobin
+      // CompactionPri)
+      c->column_family_data()->current()->storage_info()->UpdateCompactCursors(
```

Comment:
Similar problems as above. Any condition to be added here to update the compact cursors? Is this a right place to update the cursors?